pkg/index/corpus.go

```diff
@@ -675,7 +675,7 @@ func (c *Corpus) mergeBlobMeta(bm camtypes.BlobMeta) error {
 	if _, dup := c.blobs[bm.Ref]; dup {
 		panic("dup blob seen")
 	}
-	bm.CamliType = c.str(bm.CamliType)
+	bm.CamliType = schema.CamliType((c.str(string(bm.CamliType))))
```

Comment:
Not really sure if that works, can you confirm?

Reply:
converting between two string-backed types is free. they'll share the same memory.